src/mmw/js/src/analyze/templates/worksheetExport.html

```diff
@@ -0,0 +1,36 @@
+<a href="#"
+    class="analyze-worksheet-link {{ 'disabled' if started or complete or disabled }}"
+    {% if disabled %}
+        data-toggle="popover" tabindex="0"
+        data-html="true" data-container="body" role="button"
+        {# TODO Update link to technical documentation #}
+        data-content="Worksheets cannot be generated for boundary shapes. Please draw, delineate, or upload a shape. For more details, see <a href='https://wikiwatershed.org/help/model-help/mmw-tech/' target='_blank'>our technical documentation.</a>"
+        data-template="<div class='popover'><div class='pull-right' id='popover-close-button' onclick='closePopover()'><i class='fa fa-times' /></div><div class='popover-content'></div><div class='arrow'></div></div>"
+    {% endif %}
+>
+    <div class="analyze-model-name">
+        Watershed Multi-Year Worksheet
+    </div>
+    <p>
+        Generates a worksheet using a the given urban
+        area of interest, pairing it with the HUC-12s
+        it belongs to, containing Land, Stream, and
+        Water Quality analyses generated using the
+        Watershed Multi-Year Model for the HUC-12.
```

Comment:
Potentially at the end here, it should indicate what can be done with it? Or maybe just another link to the TBD-technical documentation. I was thinking something along the lines of:

> The worksheet will allow you to make more detailed BMP analyses

But truthfully, I don't have a great understanding of how the sheet is to be used.

Reply:
Let me see if I can include a link to the tech documentation somehow. Can't just pop it in because the whole box is clickable.

Reply:
I did add that line. Forewent adding the link for now. Will confer with BME once this is up to fine tune it.